src/pathfinding_service/api.py

```diff
@@ -430,6 +421,28 @@ def get(  # pylint: disable=no-self-use
                     dict(source=req["source"], target=req["target"], routes=req["routes"])
                 )
 
+        decoded_target_address: Optional[Address] = None
+        if target_address:
+            decoded_target_address = Address(to_canonical_address(target_address))
```

Comment:
Is `Address` needed here? `to_canonical_address` should return an `Address` already

Reply:
Fixed